stages/org.osbuild.firewall

```diff
@@ -92,6 +119,37 @@ def main(tree, options):
                        list(map(lambda x: f"--remove-service={x}", disabled_services)),
                        check=True)
 
+    for zone_item in zones:
+        # specifying an empty zone flag results in the source being applied to
+        # the default zone
+        zone_name = zone_item['name']
+        # check that the given zone exists, if not create it
+        if zone_name != "":
+            try:
+                subprocess.run(["chroot",
+                                tree,
+                                "firewall-offline-cmd",
+                                f"--info-zone={zone_name}"],
+                               check=True)
+            except subprocess.CalledProcessError as e:
```

Comment:
I almost would prefer to use `check=False` and check the return code, a la:

```
r = subprocess.run()

if r == 112:
   r = subprocess.run()

if r != 0:
   print(r.stderr)
```
because that stack trace is also not super helpful.

Reply:
it is not a huge deal though, if there is nothing else might as well do it in a follow up.

Reply:
Will change it, no problem :+1: